src/main/java/appeng/fluids/helper/DualityFluidInterface.java

```diff
@@ -377,7 +335,6 @@ else if( req.getFluid().equals( stored.getFluid() ) ) // same type ( qty differe
 
 	private boolean usePlan( final int slot )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The Cyclomatic Complexity of this method "usePlan" is 13 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)